west.yml

```diff
@@ -47,7 +47,7 @@ manifest:
       revision: a12d92816a53a521d79cefcf5c38b9dc8a4fed6e
       path: modules/hal/cypress
     - name: hal_nordic
-      revision: f6d5eca68e07551d6a36359b67600f9e98736839
+      revision: pull/29/head
```

Comment:
you can use the SHA here now

Reply:
@carlescufi 

OK, thanks. Done, dropped manifest changes for hal_nordic.

Reply:
@ozhuraki I meant the new SHA, see comment by @nvlsianpu